CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ FEATURES:
 * **New Resource**: `r/tfe_registry_provider` is a new resource for managing public and private providers in the private registry, by @tmatilai [1185](https://github.com/hashicorp/terraform-provider-tfe/pull/1185)
 * **New Data Source**: `d/tfe_registry_provider` is a new data source to retrieve information about a public or private provider in the private registry, by @tmatilai [1185](https://github.com/hashicorp/terraform-provider-tfe/pull/1185)
 * **New Data Source**: `d/tfe_registry_providers` is a new data source to retrieve information about public and private providers in the private registry, by @tmatilai [1185](https://github.com/hashicorp/terraform-provider-tfe/pull/1185)
+* **New Resource**: `tfe_sentinel_version` ([#1202](https://github.com/hashicorp/terraform-provider-tfe/pull/1202))
+* **New Resource**: `tfe_opa_version` ([#1202](https://github.com/hashicorp/terraform-provider-tfe/pull/1202))
```

Comment:
Should these entries match above?

Reply:
yes, since I have added support for both opa and sentinel provider in this same PR

Reply:
@mrinalirao oh the UI context is missing here. Sorry, I meant above in the files view.

https://github.com/hashicorp/terraform-provider-tfe/pull/1202/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL6

Reply:
I think it needs a `r/` prefix plus a bit more description.

Reply:
Correct! We prefix with `r/` to represent a resource and `d/` for data source in our changelog.